sw/device/lib/runtime/BUILD

```diff
@@ -0,0 +1,98 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+
+cc_library(
+    name = "epmp",
+    srcs = ["epmp.c"],
+    hdrs = ["epmp.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        "//sw/device/lib/base/freestanding",
+    ],
+)
+
+cc_library(
+    name = "hart",
+    srcs = ["hart.c"],
+    hdrs = ["hart.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        ":ibex",
+        "//sw/device/lib/arch:device",
+        "//sw/device/lib/base",
```

Comment:
Uh, do we really want these to be one target?

Reply:
This one falls under the concise/verbose tradeoff. We can always depend on specific targets to remove uncompatible functions in the future.

Reply:
For this I'd like to make another draft PR that will depend on this commit. I'm currently working on that.

Reply:
Here's a draft PR to make the individual components individually available: https://github.com/lowRISC/opentitan/pull/9144